### PR TITLE
feature/BPSINT-4-Bottom-Navigation Implementation

### PR DESCRIPTION
**Pull request by Inasity**
Description: Added bottom sheet navigation to app.

Ticket: https://bpsinterns.atlassian.net/browse/BPSINT-4

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -87,3 +87,4 @@ lint/tmp/
 
 #Api keys kept from github repo
 app/src/main/java/com/bps/gotwinter2021/common/secret/API.kt
+app/src/debug/res/values/google_maps_api.xml
```
Comment (Blu3Parr0t) [inline .gitignore]: remove this.

File: app/src/debug/res/values/google_maps_api.xml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+<resources>
```
Comment (Blu3Parr0t) [inline app/src/debug/res/values/google_maps_api.xml]: remove this.

**Review by Blu3Parr0t — CHANGES_REQUESTED** [review]
Only include things that are relative to the work of bottom navigation.

File: app/src/main/res/values/themes.xml [hunk 3]
```diff
@@ -1,8 +1,8 @@
 <resources xmlns:tools="http://schemas.android.com/tools">
     <!-- Base application theme. -->
-    <style name="Theme.GOTWinter2021" parent="Theme.MaterialComponents.DayNight.DarkActionBar">
+    <style name="Theme.GOTWinter2021" parent="Theme.MaterialComponents.DayNight.NoActionBar">
         <!-- Primary brand color. -->
-        <item name="colorPrimary">@color/purple_500</item>
+        <item name="colorPrimary">#fbcd1f</item>
```
Comment (Blu3Parr0t) [inline app/src/main/res/values/themes.xml]: Please include color in colors.xml

File: app/src/main/res/layout/fragment_home_screen.xml [hunk 4]
```diff
@@ -3,12 +3,12 @@
...
     android:layout_height="match_parent"
-    tools:context=".ui.homescreen.HomeScreen">
+    tools:context=".ui.homescreen.HomeScreenFragment">
 
     <!-- TODO: Update blank fragment layout -->
     <TextView
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:text="@string/hello_blank_fragment" />
+        android:text="Home screen fragment" />
```
Comment (Blu3Parr0t) [inline app/src/main/res/layout/fragment_home_screen.xml]: If you don't need it don't add it.

File: app/src/main/res/layout/fragment_display_screen.xml [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".ui.displayscreen.DisplayScreenFragment">
+
+    <!-- TODO: Update blank fragment layout -->
+    <TextView
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:text="Display Screen Fragment" />
```
Comment (Blu3Parr0t) [inline app/src/main/res/layout/fragment_display_screen.xml]: No hardcoded strings. If you don't need it. Don't add it.

File: app/src/main/res/layout/activity_main.xml [hunk 6]
```diff
@@ -6,13 +6,26 @@
...
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:navGraph="@navigation/main_nav" />
+
+    <com.google.android.material.bottomnavigation.BottomNavigationView
+        android:id="@+id/bottomNavigationView"
+        android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:text="Hello World!"
+        android:background="#1f1f39"
```
Comment (Blu3Parr0t) [inline app/src/main/res/layout/activity_main.xml]: No hard coded colors.

**Review by jjhuert99 — APPROVED** [review]
all looks great

**Review by Rafa311018 — APPROVED** [review]

**Review by Blu3Parr0t — APPROVED** [review]
LGTM

**Pull request merged**